<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_serverConfigurationDialog</class>
 <widget class="QDialog" name="w_serverConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia - Serwer</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>356</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLineEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>15</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>15</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="listeningOptionGroupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>50</y>
     <width>370</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Nasłuchiwanie</string>
   </property>
   <widget class="QRadioButton" name="allRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nasłuchuj na wszystkich dostępnych interfejsach</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="interfaceRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jeden interfejs</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="interfaceListGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>350</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Lista dostępnych interfejsów sieciowych</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>w_serverConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>w_serverConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
